
Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpr-build-comment[0m and container [32mstep-credential-initializer-fpktr[0m
{"level":"warn","ts":1581006088.072368,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1581006088.0731192,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpr-build-comment[0m and container [32mstep-working-dir-initializer-hw7zq[0m
{"level":"warn","ts":1581006088.554162,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581006088.5558815,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpr-build-comment[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpr-build-comment[0m and container [32mstep-create-dir-workspace-rp2rj[0m
{"level":"warn","ts":1581006094.6672196,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581006094.6697683,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpr-build-comment[0m and container [32mstep-git-source-seldonio-seldon-core-pr-1365-li-2d745[0m
{"level":"warn","ts":1581006094.9792986,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1581006098.2480319,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ master in path /workspace/source"}

Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpr-build-comment[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:9e81a0067ca9fd7a18a41b32e37e5e0e68cb076c,1365:7dcda727ff111a61c96f4daa1e791aa18404a651
DEBUG: ran git fetch --unshallow origin 7dcda727ff111a61c96f4daa1e791aa18404a651: 9e81a0067ca9fd7a18a41b32e37e5e0e68cb076c: in 
DEBUG: ran git checkout master in 
DEBUG: ran git reset --hard 9e81a0067ca9fd7a18a41b32e37e5e0e68cb076c in 
DEBUG: ran clean --force -d . in 
DEBUG: ran git merge 7dcda727ff111a61c96f4daa1e791aa18404a651 in 
Merged SHA 7dcda727ff111a61c96f4daa1e791aa18404a651 with commit message 'add break in the loop so it finish as it should at this point' into base branch master
Merged SHA 69d72af276e3444d3f21b41184d207d982531de5 with commit message 'add test that checks model which route does not accept **kwargs' into base branch master
Merged SHA c5dcc34e93d49d15fe49607c341e968373bf0ea1 with commit message 'allow route method to use metadata, closes #591' into base branch master

Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpr-build-comment[0m and container [32mstep-step2[0m
+++ dirname ./add-pr-build-comment
++ cd .
++ pwd
+ STARTUP_DIR=/workspace/source/ci
+ REPO_OWNER=SeldonIO
+ REPO_NAME=seldon-core
+ PULL_NUMBER=1365
+ BUILD_NUMBER=25
+ PIPELINE_CONTEXT=lint
++ cat
+++ date
+ COMMENT_MSG='Thu Feb  6 16:21:45 UTC 2020
The logs for [lint] [25] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1365/25.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1365 --build=25'
+ ** step pr comment --owner=SeldonIO --repository=seldon-core --pull-request=1365 '--comment=Thu Feb  6 16:21:45 UTC 2020
The logs for [lint] [25] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1365/25.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1365 --build=25'

Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpr-build-comment[0m and container [32mstep-source-mkdir-seldonio-seldon-core-pr-1365-li-zmj24[0m
{"level":"warn","ts":1581006107.8064039,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581006107.8103797,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpr-build-comment[0m and container [32mstep-source-copy-seldonio-seldon-core-pr-1365-li-2kz2s[0m
{"level":"warn","ts":1581006108.0795872,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581006108.3729274,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r source/. /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpython-lint-task[0m and container [32mstep-credential-initializer-xlt2n[0m
{"level":"warn","ts":1581006123.232467,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1581006123.233064,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpython-lint-task[0m and container [32mstep-working-dir-initializer-s2fhp[0m
{"level":"warn","ts":1581006124.232107,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581006124.234101,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpython-lint-task[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpython-lint-task[0m and container [32mstep-create-dir-workspace-7579r[0m
{"level":"warn","ts":1581006127.3225276,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581006127.3240044,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpython-lint-task[0m and container [32mstep-source-copy-workspace-c764b[0m
{"level":"warn","ts":1581006127.6397502,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581006129.8933346,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r /pvc/pr-build-comment/workspace/. /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1365-li-92bdw-25[0m stage [32mpython-lint-task[0m and container [32mstep-python-lint[0m
make: Entering directory '/workspace/source/python'
pip install black==19.3b0
Collecting black==19.3b0
  Downloading https://files.pythonhosted.org/packages/30/62/cf549544a5fe990bbaeca21e9c419501b2de7a701ab0afb377bc81676600/black-19.3b0-py36-none-any.whl (89kB)
Collecting appdirs
  Downloading https://files.pythonhosted.org/packages/56/eb/810e700ed1349edde4cbdc1b2a21e28cdf115f9faf263f6bbf8447c1abf3/appdirs-1.4.3-py2.py3-none-any.whl
Collecting attrs>=18.1.0
  Downloading https://files.pythonhosted.org/packages/a2/db/4313ab3be961f7a763066401fb77f7748373b6094076ae2bda2806988af6/attrs-19.3.0-py2.py3-none-any.whl
Collecting toml>=0.9.4
  Downloading https://files.pythonhosted.org/packages/a2/12/ced7105d2de62fa7c8fb5fce92cc4ce66b57c95fb875e9318dba7f8c5db0/toml-0.10.0-py2.py3-none-any.whl
Collecting click>=6.5
  Downloading https://files.pythonhosted.org/packages/fa/37/45185cb5abbc30d7257104c434fe0b07e5a195a6847506c074527aa599ec/Click-7.0-py2.py3-none-any.whl (81kB)
Installing collected packages: appdirs, attrs, toml, click, black
Successfully installed appdirs-1.4.3 attrs-19.3.0 black-19.3b0 click-7.0 toml-0.10.0
WARNING: You are using pip version 19.3.1; however, version 20.0.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
black \
	--check ./ ../testing \
	--exclude "(testing/scripts/proto|seldon_core/proto/|.eggs|.tox)"
All done! ‚ú® üç∞ ‚ú®
54 files would be left unchanged.
make: Leaving directory '/workspace/source/python'
